### ISSUE TITLE
Download or display better quality picture from scryfall

### ISSUE THREAD

**Issue by Porkepix** [Manuel-777/MTG-Arena-Tool#1097]
As visible in the joined screenshot, card picture can be barely legible when hovering to get the copy from scryfall. It should be displayed in a better quality.

![artisan_search](https://user-images.githubusercontent.com/709500/82790606-9f084f80-9e6c-11ea-8d2c-a627c541b35f.PNG)

**Comment by Thaoden** [comment]
What did you set the hover card size to in Settings -> Visual?

**Comment by Porkepix** [comment]
It was default sizes, but actually more problematic was the default for "Image quality" which was set to small (This is a new install from two days ago, using default values: I changed nothing there).

Maybe it could be nice to default on high quality rather than small?
As for size, calculate it from screen resolution might be good I think? Either that's already the case for size, either default changed over time: on a computer with old install and 1280x104 screen, it was set to 130px for both values, while on a new install on 2560x1440 screen, it was set (I think, from memory as I now moved those a bit) to 250px for hover size and 130px for collection card size.

**Comment by Thaoden** [comment]
So with the higher quality settings, all works as expected?

**Comment by Porkepix** [comment]
Yes, picture becomes legible. So I think this issue is actually now more about having better default values.

**Comment by Thaoden** [comment]
As for the default settings, might be better for those with smaller bandwiths or pay per byte...

**Comment by Porkepix** [comment]
> As for the default settings, might be better for those with smaller bandwiths or pay per byte...

That's right, but is this still the case for most people in the World? From a user standpoint, that's not obvious this is a setting you can and have to change to have the picture in normal quality. Or maybe have a way to offer the choice to user on the first start?

**Comment by Thaoden** [comment]
You're welcome to add some [docs](https://github.com/mtgatool/mtgatool-web/tree/master/src/components/docs) ;)
Personally, I like defaults to be on the defensive side. And the setting is there, it's not hidden away under 5 layers of submenus or anything.

**Comment by Porkepix** [comment]
> You're welcome to add some [docs](https://github.com/mtgatool/mtgatool-web/tree/master/src/components/docs) ;)

I might have try… if it was some simple Markdown or other markup language/text file, but here it seems that the doc is also some kind of TypeScript.

> Personally, I like defaults to be on the defensive side. And the setting is there, it's not hidden away under 5 layers of submenus or anything.

You're right it's not hidden. However it's not obvious that it's a setting you can change, if it was then I would have check out the settings first rather than opening an issue. That's only after you pointed me to the section that I went and saw those (I might have seen it like one year ago and forget about it, but that doesn't change my point).

That's why offering a choice to user on first start might be an idea. As for the defensive side, right, but while it can cause a few users to use a little bit more data temporarily and allow them to fix this in a timely manner, this also would, I think, make better defaults for most of the users.
I can obviously be wrong on the proportions of users that would pick small or high picture quality as that's by a rule of thumb.

**Comment by Thaoden** [comment]
It's markdown, afaik.

Honestly, I don't get your point. Take any other piece of software. There is something you don't like with the layout. You go into the settings panel and search for the corresponding setting. Next thing is you web seach for it (granted, this is not yet documented, so web search wouldn't have helped here).

But this is just a single standpoint too, so we might need more data ;)

**Comment by Porkepix** [comment]
> It's markdown, afaik.

It's my bad, I looked at many places in the repository, but not in the single directory from your direct link… <_<
And yeah, that's Markdown.

> Honestly, I don't get your point. Take any other piece of software. There is something you don't like with the layout. You go into the settings panel and search for the corresponding setting. Next thing is you web seach for it (granted, this is not yet documented, so web search wouldn't have helped here).
> 
> But this is just a single standpoint too, so we might need more data ;)

Honestly, it depends a lot on the software, and on what you might expect to configure or not. Or the platform.
For example, if Tool was a mobile application, I would have expected a lot more to find such setting, not in a desktop environment. I would expect a lot more to be able to change date and time format, for example, theme and colors, presentation format, or that annoying thin border on the overlays, for example :p

Also, I recently reported in a Reddit reader application similar issue of blurred pictures. It was a bug in one of their libs in their case.
